.github/workflows/ci.yml

```diff
@@ -126,7 +126,7 @@ jobs:
       matrix:
         terraform:
           - '1.8.*'
```

Comment:
We can remove 1.8 as its EOL

<img width="826" alt="Screenshot 2024-12-10 at 8 55 11 AM" src="https://github.com/user-attachments/assets/cd58fbc6-4033-4b69-a296-ce885e5ee366">

Reply:
good point